zebra/interface.c

```diff
@@ -4853,17 +4853,14 @@ static int ag_iter_cb(const struct lyd_node *dnode, void *arg)
 void cli_show_legacy_admin_group(struct vty *vty, const struct lyd_node *dnode,
 				 bool show_defaults)
 {
-	if (!yang_dnode_exists(dnode, "./legacy-admin-group"))
-		return;
-
 	vty_out(vty, "  admin-group 0x%x\n",
-		yang_dnode_get_uint32(dnode, "./legacy-admin-group"));
+		yang_dnode_get_uint32(dnode, NULL));
 }
 
 void cli_show_affinity_mode(struct vty *vty, const struct lyd_node *dnode,
 			    bool show_defaults)
 {
-	enum affinity_mode affinity_mode = yang_dnode_get_enum(dnode, ".");
```

Comment:
so ... I think affinity-mode is just busted:
a) this doesn't handle the EXTENDED value at all
b) since the value zero is "STANDARD", it doesn't look like you can remove this with "no"?